pattoo/api/portal/react/src/AgentsRow.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+
+import Modal from './Modal'; // Used to display datapoints for agent
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.